core/controllers/ingest.py

```diff
@@ -51,28 +52,39 @@ def ingest(body: dict, excel_file: FileStorage) -> Response:
     validate(workbook, original_workbook, reporting_round)
 
     clean_data(workbook)
+
+    if not do_not_load:
+        load_data(workbook, excel_file, reporting_round)
+
+    success_payload = {
+        "detail": f"Spreadsheet successfully validated{' and ingested' if not do_not_load else ''}",
```

Comment:
Do we not need to make it clear to the user that the spreadsheet has definitely not been ingested? Maybe saying it has been "validated" is a little bit ambiguous. I suppose we need to change the frontend screen to make this clear.

Reply:
The user will never see this - this is just the API response but yeah it's a fair point for even this message

Reply:
Added the `loaded` attribute to the JSON response for further clarity:

https://github.com/communitiesuk/funding-service-design-post-award-data-store/blob/e8c78b9c70f5d83d1056bd365804d05f4038aa2b/core/controllers/ingest.py#L59-L65